web/src/components/start_here/subcomponents/before_the_event.js

```diff
@@ -0,0 +1,129 @@
+import {
+  Box,
+  Container,
+  Link,
+  List,
+  ListItem,
+  Typography,
+} from "@mui/material";
+import messages from "../messages";
+import startHereStyles from "../start_here.styles";
+import FeatureFlags from "infra/feature_flags";
+
+function Text(props) {
+  return <Typography display="inline">{props.children}</Typography>;
+}
+
+function Space() {
+  return <Text>&nbsp;</Text>;
+}
+
+function Review() {
+  let showCodeOfConduct = FeatureFlags.showCodeOfConductTab;
+  let showHealthPolicy = FeatureFlags.showHealthTab;
+
+  return (
+    <Box>
+      <Text>{messages.beforeTheEvent.review.our}</Text>
+      {!!showCodeOfConduct && [
+        <Space />,
+        <Link href="#/code" target="__blank" rel="noopener noreferrer">
+          <Text>{messages.beforeTheEvent.review.codeOfConduct}</Text>
+        </Link>,
+      ]}
+      {!!(showCodeOfConduct && showHealthPolicy) && [
+        <Space />,
+        <Text>{messages.beforeTheEvent.review.and}</Text>,
+      ]}
+      {!!showHealthPolicy && [
+        <Space />,
+        <Link href="#/health" target="__blank" rel="noopener noreferrer">
+          <Text>{messages.beforeTheEvent.review.healthPolicy}</Text>
+        </Link>,
+      ]}
+    </Box>
+  );
+}
+
+function RegistrationForm() {
+  return (
+    <Box>
+      <Text>{messages.beforeTheEvent.registration.fillOut}</Text>
+      <Space />
+      <Link
+        href="https://forms.gle/2kbGxh399PQ2seEe7"
+        target="__blank"
+        rel="noopener noreferrer"
+      >
+        <Text>{messages.beforeTheEvent.registration.registrationForm}</Text>
```

Comment:
Do we have this requirement anymore? Since we're moving away from vax required, are we formally registering everyone who comes?

Reply:
No idea. We can ask Riley and remove it later